docs/docs/getting-started/apps.md

```
@@ -72,94 +72,162 @@ curl "http://192.168.0.136:53287/brightness?value=0.5"
-    adb install -r /path/to/ImmichFrame_vXX.apk
+
+2. Start Frameo app:
+    ```bash 
+    adb shell am start net.frameo.frame
     ```
-  - Start ImmichFrame:
-    ```bash
-    adb shell am start com.immichframe.immichframe/.MainActivity
+
+3. Uninstall ImmichFrame:
+    ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)